### PR TITLE
Add validation that .html and .jsonld files exist alongside with standard

### PR DESCRIPTION
**Pull request by fbjorn**

### CODE REVIEW

File: openapi_to_fastapi/validator/ihan_standards.py [hunk 1]
```diff
@@ -109,7 +124,30 @@ def validate_spec(spec: dict):
         raise AuthProviderHeaderMissing
 
 
+def check_extra_files_exist(path: Path):
+    html = path.parent / path.name.replace(path.suffix, ".html")
```
Comment (joakimnordling) [inline openapi_to_fastapi/validator/ihan_standards.py:128]:
```suggestion
    html = path.with_suffix(".html")
```

File: openapi_to_fastapi/tests/test_ihan_standards.py [hunk 2]
```diff
@@ -133,3 +135,29 @@ def test_auth_provider_header_is_missing(tmp_path):
...
     spec["paths"]["/Company/BasicInfo"]["post"]["parameters"] = [auth_header]
     check_validation_error(tmp_path, spec, ihan.AuthProviderHeaderMissing)
+
+
+def test_missing_html_file(tmp_path):
+    spec = deepcopy(COMPANY_BASIC_INFO)
+    spec_path = tmp_path / "spec.json"
+    spec_path.write_text(json.dumps(spec))
+    (tmp_path / "spec.jsonld").write_text('{"a":"b"}')
+    with pytest.raises(ihan.StandardComponentMissing):
```
Comment (joakimnordling) [inline openapi_to_fastapi/tests/test_ihan_standards.py:145]: Should we also test with empty content in the file?

Comment (fbjorn) [reply openapi_to_fastapi/tests/test_ihan_standards.py:145]: added

**Review by joakimnordling — APPROVED** [review]
LGTM
